src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--tue calendar--month-length--28">
+      <div class="cell calendar__cell"><span class="p-1"></span></div>
```

Comment:
I think tag span is not required here, You could have use modifier. 
Like-->  calendar--cell-no--1

Reply:
Fixed